src/GraphQL.ApiTests/GraphQL.SystemTextJson.approved.txt

```diff
@@ -30,9 +18,40 @@ namespace GraphQL.SystemTextJson
         public static GraphQL.DI.IGraphQLBuilder AddSystemTextJson(this GraphQL.DI.IGraphQLBuilder builder, System.Action<System.Text.Json.JsonSerializerOptions>? action = null) { }
         public static GraphQL.DI.IGraphQLBuilder AddSystemTextJson(this GraphQL.DI.IGraphQLBuilder builder, System.Action<System.Text.Json.JsonSerializerOptions, System.IServiceProvider>? action) { }
```

Comment:
Let's comment all additional methods for now.

Reply:
> Let's comment all additional methods for now.

I'm sorry, do what?

Reply:
comment:

blabla   ->    // blabla

> So then perhaps it is pointless to offer the specific ReadNode<T>(JsonElement) and ReadNode<T>(JsonDocument) methods on the class. We probably don't need either method because of the IGraphQLSerializer.ReadNode(object) method

Agree. Uncomment further if/when needed.